apps/ai/schemas/file_analysis.py

```
@@ -1,92 +1,69 @@
-
-    total_rows: int
-    total_columns: int
+    total_rows: int = Field(ge=0)
+    total_columns: int = Field(ge=0)
     primary_candidates: PrimaryCandidates
 
-class Warning(BaseModel):
-
-    """소스 레벨 경고입니다."""
 
+class Warning(BaseModel):
```

Ruff (0.15.12)

[error] Variable `Warning` is shadowing a Python builtin

(A001)